index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo-List</title>
+    <link rel="stylesheet" href="./index.css" />
+    <script src="https://kit.fontawesome.com/043c3d901b.js" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Sortable/1.15.0/Sortable.min.js" integrity="sha512-Eezs+g9Lq4TCCq0wae01s9PuNWzHYoCMkE97e2qdkYthpI0pzC3UGB03lgEHn2XM85hDOUF6qgqqszs+iXU4UA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+  </head>
+  <body>
+    <form class="todo">
+      <div class="todo-list-title">
+        <p> Todo List</p>
+        <i id="icon" class="fas fa-sync recycle"></i>
+      </div>
+      <input type="text" class="activity" placeholder="Add to your list..." />
+      <ul class="todo-list"></ul>
+      <button type="button" class="clear bg-color border-0">Clear all completed</button>
+    </form>
+    <script type= "module" src="./index.js">
+
```

Comment:
Nice work on your code so far so Good. However, I suggest that you divide your code using HTML semantics for ease of accessibility: like having a header, a footer, and may be a main in-between.
For the input also add A label but hidden.
This is so that your site can easily be accessed by visually impaired persons and those who use computer aided assistance